src/commands/executeFunction/eventGrid/EventGridCodeLensProvider.ts

```diff
@@ -0,0 +1,22 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { CodeLens, Range, type CodeLensProvider, type TextDocument } from 'vscode';
+import { localize } from '../../../localize';
+
+export class EventGridCodeLensProvider implements CodeLensProvider {
+    public provideCodeLenses(document: TextDocument): CodeLens[] {
+        const firstLineLens = new CodeLens(new Range(0, 0, 0, 0));
+        const lastLine = document.lineAt(document.lineCount - 1);
```

Comment:
As discussed offline, let's just remove this and have it at the top only.

Reply:
For reference, this is what it looks like: 
![image](https://github.com/microsoft/vscode-azurefunctions/assets/5290572/3567452a-0a61-43e8-b61b-06a6dba5ac2c)